CHANGELOG.md

```diff
@@ -4,10 +4,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.4.0 - 2022-04-13
+### Added
+- A project resolver to decide which project is installed. Also added a config to overwrite the type used.
+- Composer requirement which should have already been there.
```

Comment:
You could be more explicit here, as in 

Fixed (we did not add a feature, we fixed a problem)
- Added missing composer dependency to package X